test/expectations/hover/documented_class.exp.json

```diff
@@ -9,16 +9,6 @@
         "contents": {
             "kind": "markdown",
             "value": "```ruby\nBar\n```\n\n**Definitions**: [fake.rb](file:///fake.rb#L2,1-3,4) | [fake.rb](file:///fake.rb#L6,1-7,4)\n\n\n\nThis is the documentation for Bar\n\nThis is more documentation for Bar"
-        },
-        "range": {
-            "start": {
-                "line": 1,
-                "character": 6
-            },
-            "end": {
-                "line": 1,
-                "character": 9
-            }
```

Comment:
I didn't expect the changes to affect the request's behaviour. Do you know what's causing these, and why are they necessary for the task?

Reply:
Really good question.

If you take a look at [this excerpt](https://github.com/Shopify/ruby-lsp/pull/1317/files#diff-ffcc875e5a79593a72a08918054a81f51e7410fb0dc451b01db8067c5814a0d5L83-L88), we were previously ignoring the ranges of all other response except the first response.

In essence, we weren't concatenating the ranges in a meaningful fashion. Given that the range itself is optional and that we aren't using it in the intended way, @vinistock and I figured it would make sense to omit it as part of the initial refactoring.

Reply:
Thanks for the explanation 👍

Reply:
We were also using `range` incorrectly. The [spec](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_hover) says it's an optional range used to visualize a hover, but we were always just passing the location of the declaration, which doesn't sound entirely correct.

Removing it seems to make no difference in the behaviour we see in the editor.